extensions/amp-analytics/0.1/variables.js

```diff
@@ -220,7 +220,7 @@ export class VariableService {
         // Expand string values further.
         p = this.expandTemplate(raw,
             new ExpansionOptions(options.vars, options.iterations - 1,
-                options.noEncode));
+                true /* noEncode */));
```

Comment:
```html
<amp-analytics>
    <script type="application/json" type="googleanalytics">
        {
          "vars": {
            "account": "UA-XXXXXX-YY"
          },
          "triggers": {
            "trackPageview": {
              "on": "visible",
              "request": "pageview"
            }, 
            "trackOutboundClick": {
                "on": "click",
                "selector": "[href*='/offsite/']",
                "request": "event",
                "vars": {
                    "eventCategory": "outbound-link-aff", 
                    "eventAction": "${outboundLink}"
                }
            }
          }
        }
    </script>
</amp-analytics>
```

The tests aren't testing any with `noEncode` set to `true.` We need a test that `outboundLink` is encoded once and `eventAction` is equal to `outboundLink`.